README.md

```diff
@@ -1,4 +1,9 @@
 # ethgasstation
+
+## Devops URLs
+- [🤖 CodeBuild staging](https://us-west-1.console.aws.amazon.com/codesuite/codebuild/017964463726/projects/defipulse-egs-legacy-backend-staging/)
+- [🐮 Rancher pods staging](https://rancher.concourseq.net/p/c-25wt9:p-sgbsd/workload/deployment:egs-legacy-staging:egs-legacy-backend)
```

Comment:
Love the idea of including important URLs like this in the README for private repos. Given this is public though I'd prefer if we did not add them.

Reply:
Good point, I brain farted thinking this was private initially, then forgot to revert that piece.
Addressed the comment